.github/workflows/stale-issues.yml

```diff
@@ -0,0 +1,43 @@
+name: "Close stale issues"
+
+# Controls when the action will run.
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  cleanup:
+    runs-on: ubuntu-latest
+    name: Stale issue job
+    steps:
+      - uses: aws-actions/stale-issue-cleanup@v3
+        with:
+          # Targeted types
+          issue-types: issues
+
+          # Issue timing
+          days-before-stale: 60
+          days-before-close: 14
+          days-before-ancient: 365
+
+          # Labels
+          stale-issue-label: closing-soon
+          exempt-issue-label: bug,documentation,feature-request,guidance,protocol-test
```

Comment:
Shouldn't close `investigating`.

I think `guidance` will need to wait to come off this list until we've either tagged them as `documentation` or chosen that they need no further update.

Reply:
I've labeled everything with a `guidance` label that needs more work with another label representing that work. The `guidance` label can be removed from this list now and we can move forward.